cibuildwheel/typing.py

```diff
@@ -9,7 +9,6 @@
     "PLATFORMS",
     "PathOrStr",
     "PlatformName",
-    "PLATFORMS",
```

Comment:
Ruff would have just moved this higher up, but I noticed that it actually appears twice in `__all__` here, so I removed the duplicate entry!